### DISCUSSION TITLE
How to change wakeup-interval- Photoprism plugin on OpenMediaVault

### DISCUSSION THREAD

**Discussion by shopboy3** [photoprism/photoprism#3294] [Q&A]
I have Photoprism running on my OMV nas using the plugin, not docker container. Install was super easy , but because it is a plugin I cannot figure out how to change the wakeup-interval. Is this possible with my setup?

**Comment by graciousgrey** [comment]
Maybe another OMV user can help. Otherwise we recommend that you ask the [openmediavault community](https://forum.openmediavault.org/) for advice. They might be either able to add the config option to their UI or explain you how to edit [config options](https://docs.photoprism.app/getting-started/config-options/) on OMV .

**Comment by mostofthetime** [comment]
The Photoprism Plugin is using the docker image with podman.
You can edit the service and add the "-e PHOTOPRISM_WAKEUP_INTERVAL=<your intervall>" to ExecStart with command "systemctl edit container-photoprism-app.service" and "systemctl daemon-reload".

Alternative you can edit the "/etc/systemd/system/container-photoprism-app.service" directly but is not recommended. Afterwards you need to restart the podman container.

To check if it works use "podman exec photoprism-app photoprism config | grep wakeup-interval"

**Reply by shopboy3** [reply]
Thank you for the response. No luck with the above solution though, probably because I am somewhat unfamiliar with podman and plugin type dockers. Using WeTTY and OMV, I was unable to edit the service. More research needed on my part.
